spec/roman_numerals_spec.rb

```diff
@@ -0,0 +1,97 @@
+require 'roman_numerals'
+
+RSpec.describe RomanNumerals do
+  roman_numeral_converter = RomanNumerals.new
+
+  it "converts 0 to empty string" do
+    roman_numeral = roman_numeral_converter.translate(0)
```

Comment:
is there any added value in having this temp variable? maybe you could rename your roman_numeral_converter variable, or the translate method name instead of requiring a temp var.